androidApp/build.gradle.kts

```diff
@@ -1,15 +1,17 @@
 plugins {
     id("com.android.application")
     id("org.jetbrains.kotlin.multiplatform")
+    // TODO: migrate to View Binding (https://goo.gle/kotlin-android-extensions-deprecation)
```

Comment:
I'm pretty sure we are already using View Bindings, so this already may be an unneeded plugn.